### PR TITLE
delay on easy reference popover

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?243816

### CODE REVIEW

File: src/richText.js [hunk 1]
```diff
@@ -749,8 +749,8 @@ define([
                     (isFormRef ? '<div class="popover-content"><p></p></div>' : '') +
                     '</div></div>',
                 delay: {
-                    show: 0,
-                    hide: 200,
+                    show: 350,  // be less annoying
```
Comment (emord) [inline src/richText.js]: Slightly related: Does this mean that the analytics event is fired after 350 ms instead of instantaneously on hover?

**Comment by orangejenny** [comment]
fixed tests

**Comment by orangejenny** [comment]
@millerdev Do you remember why there's a `try` in those tests in the first place? Wondering if I should assume that an exception means the test failed, and then assert false inside the `catch`.

**Comment by millerdev** [comment]
I think the `try`/`finally` is important for the popover to be removed, otherwise other tests that test popovers will fail.

**Comment by orangejenny** [comment]
Fixed tests for real.

**Pull request merged**